features/visitor_menu.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Visitor can see the menu
```

Comment:
Remove this test, we only need one feature file

Reply:
Incorporated into display_menu_item.feature